### DISCUSSION TITLE
How should I load NeuralMagic Nous-Hermes Solar into Webui?

### DISCUSSION THREAD

**Discussion by forpr1093** [oobabooga/text-generation-webui#5371] [Q&A] [answered]
I want to try loading this model: https://huggingface.co/neuralmagic/Nous-Hermes-2-SOLAR-10.7B-pruned50-quant-ds

But there is no model loader able to load the model.

Any help would be appreciated!

**Comment by araleza** [comment, marked as answer]
I don't think webui supports DeepSparse, so you won't be able to load that model, at least for now.  Instead, if you want a GGUF version (which can be loaded by ooba webui, and also can be run on a CPU) of a similar Nous-Hermes-2-SOLAR-10.7B model, you could grab a GGUF from TheBloke:

https://huggingface.co/TheBloke/Nous-Hermes-2-SOLAR-10.7B-GGUF
